doc/source/release/1.18.0-notes.rst

```diff
@@ -1,8 +1,348 @@
-The NumPy 1.18 release is currently in development. Please check
-the ``numpy/doc/release/upcoming_changes/`` folder for upcoming
-release notes.
-The ``numpy/doc/release/upcoming_changes/README.txt`` details how
-to add new release notes.
-
-For the work in progress release notes for the current development
-version, see the `devdocs <https://numpy.org/devdocs/release.html>`__.
+================================
+NumPy NumPy 1.18.0 Release Notes
+================================
+
+
+New functions
+=============
+
+Multivariate hypergeometric distribution added to `numpy.random`
+----------------------------------------------------------------
+The method `multivariate_hypergeometric` has been added to the class
+`numpy.random.Generator`.  This method generates random variates from
+the multivariate hypergeometric probability distribution.
+(`gh-13794 <https://github.com/numpy/numpy/pull/13794>`__)
+
+
+Deprecations
+============
+
+`np.fromfile` and `np.fromstring` will error on bad data
+--------------------------------------------------------
+
+In future numpy releases, the functions `np.fromfile` and `np.fromstring`
+will throw an error when parsing bad data.
+This will now give a ``DeprecationWarning`` where previously partial or
+even invalid data was silently returned. This deprecation also affects
+the C defined functions c:func:`PyArray_FromString`` and
```

Comment:
I'd much prefer double backticks over `c:func:`, because we're using these release notes in other places (like https://github.com/numpy/numpy/releases and in the Tidelift dashboard) where `c:func:` will just look weird.

Reply:
I've gone ahead and changed all the links to double backticks for consistency. If links are preferred the fix will not be difficult. I think we can now use `np.*` as well as `numpy.*`. @mattip?

Reply:
> because we're using these release notes in other places (like https://github.com/numpy/numpy/releases 

Perhaps we should enhance the [pavement.py script] to do more. I don't even see where it converts rst to markdown, but it is writing a file with a `md` suffix. @charris do you need to massage this by hand?

> we can now use `np.*` as well as `numpy.*`

Unfortunately no, we have not worked out a way to do that. [Here](https://10526-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/release/1.18.0-notes.html) is a rendered release note from a PR, the `np.*` links are broken. I tried to fix that in gh-11344, but failed, maybe now that we have moved to sphinx 2.2.0 there is a way to support that